db/log_reader.cc

```diff
@@ -130,13 +132,16 @@ bool Reader::ReadRecord(Slice* record, std::string* scratch,
         prospective_record_offset = physical_record_offset;
         scratch->assign(fragment.data(), fragment.size());
         in_fragmented_record = true;
+        skipped_ = false;
```

Comment:
Need to also clear after `kUserDefinedTimestampSizeType`.

Reply:
done, thx for review